content/_posts/falix/dashboard/2024-10-18-create-server.md

```diff
@@ -6,30 +6,87 @@ description: A step by step article to create your very first server.
 keywords:
     - keyword: server
       matches: ["create", "make"]
-author: Mocab
+author: 
+      - Mocab
+      - Cakey
 toc: false
 
 getting-started-tag: General
 post_order: 1
 ---
 
+
+This guide walks you through the updated process of creating a server on FalixNodes using the latest dashboard.
+
+## Creating the Server
+
 1. Log in to the [Dashboard](https://client.falixnodes.net/).
 
 2. In the navigation menu, click on [Create Server](https://client.falixnodes.net/create).
 
-3. Scroll down to "Create Server" and input the respective values into each field. Please refer to the table below:
+3. Scroll down to the **Plan Selection** section.
+
+4. Choose either:
+   - **Free**	
+   - **Premium**
+
+5. Click **Continue** to proceed.
+   
+### Enter Server Details
+
+On this page, fill in the following details:
+
+- **Server Name**  
+
+    - `A name used to identify your server in the dashboard. This is only visible to you and can be changed later.`
+
+- **Server Domain**  
+
+    - 	`The address players will use to connect to your server. This can also be changed later.	`
+
+Once completed, click **Continue**.
+
+## Select the Game or Application
+
+1. Choose the game you want to run.
+   - For most users, select **Minecraft**.
+   - Premium users may also select other applications from 	`Applications` tab such as **Node.js** or **Python**.
```

Comment:
Other games should also be clarified, as we support much more than just Minecraft